DetectorDescription/DDCMS/plugins/DDTECModuleAlgo.cc

```diff
@@ -80,14 +43,17 @@ static long algorithm(Detector& /* description */, cms::DDParsingContext& ctxt,
   double topFrame2LHeight = isStereo ? args.value<double>("TopFrame2LHeight") : 0e0;  //             left  height
   double topFrame2RHeight = isStereo ? args.value<double>("TopFrame2RHeight") : 0e0;  //             right height
   double topFrameZ = args.value<double>("TopFrameZ");                                 //              z-positions
-  string sideFrameMat = args.value<string>("SideFrameMaterial");                      //Side frame    material
-  double sideFrameThick = args.value<double>("SideFrameThick");                       //              thickness
+
+  double resizeH = 0.96;
```

Comment:
@vargasa @ianna sorry what is this magic number? I see it used as a scaling factor, as it is hardcoded a comment could be helpful

Reply:
@fabiocos I felt the same discomfort but It has been there since the beginning of the universe:

https://github.com/cms-sw/cmssw/blob/214ab7313d6eb5aca58c29f7cdfce0df3a223667/Geometry/TrackerCommonData/plugins/DDTECModuleAlgo.cc#L79

Reply:
@vargasa right, but as we (well you) are now touching again this code, it may be an opportunity to leave a more explicative message for the posterity (and the poor code reviewer of the future)...

Reply:
@fabiocos - the scale might have been introduced to remove overlaps. Unfortunately, any comment or explanation would be a guess at this stage.